### PR TITLE
Revert - PR 2531

### PR DESCRIPTION
**Pull request by raghupathiguduri**
https://github.com/project-sunbird/sunbird-devops/pull/2531
Moving form read to short validity

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Enhancement (additive changes to improve performance)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by keshavprasadms — APPROVED** [review]

**Comment by keshavprasadms** [comment]
@rjshrjndrn please review monitoring changes

File: ansible/cassandra-exporter.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-- hosts: cassandra
+- hosts: cassandra-ps
```
Comment (rjshrjndrn) [inline ansible/cassandra-exporter.yaml:1]: why cassandra-ps ?

Comment (rjshrjndrn) [reply ansible/cassandra-exporter.yaml:1]: Can you please name it as cassandra-monitoring-hosts. It'll make the group name self explanatory.

Comment (raghupathiguduri) [reply ansible/cassandra-exporter.yaml]: Updated the host name as mentioned @rjshrjndrn

**Review by rjshrjndrn — APPROVED** [review]

File: kubernetes/ansible/roles/sunbird-monitoring/templates/additional-scrape-configs.yaml [hunk 2]
```diff
@@ -52,7 +52,7 @@ scrapeconfig:
   - job_name: 'cassandra-exporter'
     metrics_path: /metrics
     static_configs:
-      - targets:  ["{{ groups['cassandra'] | difference(["localhost"]) | map('regex_replace', '^(.*)$', '\\1:8080' ) | list | join("\", \"") }}"]
+      - targets:  ["{{ groups['cassandra-ps'] | difference(["localhost"]) | map('regex_replace', '^(.*)$', '\\1:8080' ) | list | join("\", \"") }}"]
```
Comment (rjshrjndrn) [inline kubernetes/ansible/roles/sunbird-monitoring/templates/additional-scrape-configs.yaml:55]: I think you'll have to change it here also @raghupathiguduri

**Review by rjshrjndrn — CHANGES_REQUESTED** [review]

**Pull request closed without merging**